packages/formation-react/src/components/AccessibleLink/AccessibleLink.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import Telephone, { CONTACTS, PATTERNS } from '../Telephone/Telephone';
+import MediaLink, { MEDIA_SITES } from '../MediaLink/MediaLink';
+import DownloadLink, { FILE_TYPES } from '../DownloadLink/DownloadLink';
+import { createIcon } from '../../helpers/link-utils';
+
+const AccessibleLink = props => {
+  const { children, icon, ...linkProps } = props;
+  return (
+    <a {...linkProps}>
+      {icon && typeof icon === 'string' ? createIcon(icon) : icon}
+      {children}
+    </a>
+  );
+};
+
+AccessibleLink.Telephone = Telephone;
+AccessibleLink.Download = DownloadLink;
+AccessibleLink.Media = MediaLink;
```

Comment:
This pattern isn't used elsewhere, but I thought it might be nice to make HOC that gives you access to all link-type components. They are all sort of related. If you think this isn't necessary, I can remove it.

Reply:
Yeah, let's not add this new pattern yet.